src/app/click/click.html

```diff
@@ -0,0 +1,7 @@
+    <div id="mainPopup">
+      <span>{{ test1 }}</span>
+      <h1>Firebase test!</h1>
+      <p>Clicks: <span id="contents">Loading...</span></p>
```

Comment:
Sky is the limit- just wanted to get something pushed... figured we'd be moving away from 'clicks' soon-ish